tests/unit_tests/test_pages.py

```diff
@@ -0,0 +1,6 @@
+def test_set_locale(app, client):
+    response = client.get("/locale/cy", headers={"referer": "http://localhost/privacy"})
+    assert response.status_code == 302
+    assert response.headers["location"] == "/privacy"
+    cookies = response.headers["Set-Cookie"].split(";")
+    assert "locale=cy" in cookies, "Could not find locale cookie"
```

Comment:
Should we add test for 404 and a playwright e2e test for this functionality?

Reply:
Added, thanks

Reply:
Added playwright tests to cover changing languages using link in the footer as well as checking the `lang` attribute on the html element